tests/shared/src/test/scala/zio/schema/DiffSpec.scala

```diff
@@ -11,23 +11,23 @@ import zio.schema.SchemaGen.{ Arity1, Arity24 }
 import zio.schema.StandardType._
 import zio.schema.syntax._
 import zio.schema.types.Arities._
-import zio.test._
+import zio.test.{ Gen, _ }
 
 object DiffSpec extends DefaultRunnableSpec {
 
   def spec: ZSpec[Environment, Failure] = suite("Differ")(
     suite("standard types")(
       suite("unit")(
-        testM("always identical") {
+        test("always identical") {
           check(Gen.unit <*> Gen.unit) {
```

Comment:
Interesting implication of the compositional zips. Can't say it is wrong since there is only one unit value. 😃

Reply:
Could just generate the unit value once. Also not sure this needs to be a property based test at all.

Reply:
Have done so now, also not entirely sure why this is a property based check so I've left it as is.